.drone.star

```diff
@@ -51,35 +51,31 @@ def testing(ctx):
             },
             {
                 'name': 'dependencies',
-                'image': 'owncloudci/nodejs:14',
-                'pull': 'always',
+                'image': 'owncloudci/nodejs:16',
```

Comment:
v16 won't be LTS for next 3 months... I'd suggest to stay with v14 till then

Reply:
Yeah I remember we had this discussion before 🥴 but since I want to use `.replaceAll()` which isn't supported in v14, but was supported in the now unsupported v15 and is supported in v16...idk, man

Reply:
You can use `.replace(regexp, ...)` to replace all.

Reply:
Resolving this, thanks for the insights 🙌🏽